presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -229,12 +229,10 @@ class SystemConfig : public ConfigBase {
   /// NOTE: this config only applies if the memory arbitration has been enabled.
   static constexpr std::string_view kMemoryPoolTransferCapacity{
       "memory-pool-transfer-capacity"};
-  /// The percentage of memory pool capacity reserved for system usage such as
-  /// the disk spilling memory usage.
-  ///
-  /// NOTE: this config only applies if the memory arbitration has been enabled.
-  static constexpr std::string_view kReservedMemoryPoolCapacityPct{
-      "reserved-memory-pool-capacity-pct"};
+  /// Enables the memory usage tracking for the system memory pool such as disk
+  /// spilling.
```

Comment:
/// Enables the memory usage tracking for the system memory pool used for cases such as disk spilling.